### ISSUE TITLE
Spyder console issue

### ISSUE THREAD

**Issue by ManlikeDBreezy** [spyder-ide/spyder#13002]
### What steps will reproduce the problem?

I am really new at this, so apologies if this is pretty dumb.

when the console loads it keeps repeating a message from a previous bit of code I was working on, even if I close the console and loan another or trying to clear the console the printed message will appear. 

I have uninstalled Anaconda and updated to the latest version of Spyder but stil can figure it out.

## Versions

* Spyder version: 4.1.3 
* Python version: 3.7.7
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0           :  1.4.0 (OK)
chardet >=2.0.0                :  3.0.4 (OK)
cloudpickle >=0.5.0            :  1.4.1 (OK)
diff_match_patch >=20181111    :  20181111 (OK)
intervaltree                   :  None (OK)
IPython >=4.0                  :  7.13.0 (OK)
jedi =0.15.2                   :  0.15.2 (OK)
nbconvert >=4.0                :  5.6.1 (OK)
numpydoc >=0.6.0               :  0.9.2 (OK)
paramiko >=2.4.0               :  2.7.1 (OK)
parso =0.5.2                   :  0.5.2 (OK)
pexpect >=4.4.0                :  4.8.0 (OK)
pickleshare >=0.4              :  0.7.5 (OK)
psutil >=5.3                   :  5.7.0 (OK)
pygments >=2.0                 :  2.6.1 (OK)
pylint >=0.25                  :  2.5.2 (OK)
pyls >=0.31.9;<0.32.0          :  0.31.10 (OK)
qdarkstyle >=2.8               :  2.8.1 (OK)
qtawesome >=0.5.7              :  0.7.0 (OK)
qtconsole >=4.6.0              :  4.7.4 (OK)
qtpy >=1.5.0                   :  1.9.0 (OK)
rtree >=0.8.3                  :  0.9.4 (OK)
sphinx >=0.6.6                 :  3.0.4 (OK)
spyder_kernels >=1.9.1;<1.10.0 :  1.9.1 (OK)
watchdog                       :  None (OK)
zmq >=17                       :  18.1.1 (OK)

# Optional:
cython >=0.21                  :  0.29.17 (OK)
matplotlib >=2.0.0             :  3.1.3 (OK)
numpy >=1.7                    :  1.18.1 (OK)
pandas >=0.13.1                :  1.0.3 (OK)
scipy >=0.17.0                 :  1.4.1 (OK)
sympy >=0.7.3                  :  1.5.1 (OK)
```

**Comment by andfoy** [comment]
Hi @ManlikeDBreezy, thanks for contacting us, could you please provide a GIF that shows the aforementioned behaviour?

**Comment by ManlikeDBreezy** [comment]
Hi, sorry for the delay.

Hopefully this GIF works.

![Spyder](https://user-images.githubusercontent.com/65815272/84562109-b6d74300-ad49-11ea-99d0-3fc8620de0c4.gif)

**Comment by ManlikeDBreezy** [comment]
Hi, I have now provided a GIF on the issue.

Thanks

Sent from my iPhone

On 11 Jun 2020, at 22:01, Edgar Andrés Margffoy Tuay <notifications@github.com> wrote:

﻿

Hi @ManlikeDBreezy<https://eur02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FManlikeDBreezy&data=02%7C01%7C%7C7acb6d3f398a4d63d07108d80e4a947f%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637275060758792055&sdata=XtFOFNmS2gfuPFAbXbG5Iwph6Dz0r0LDtE9RXqeErlo%3D&reserved=0>, thanks for contacting us, could you please provide a GIF that shows the aforementioned behaviour?

**Comment by bcolsen** [comment]
Inputs can cause some problems.

@ManlikeDBreezy Do you have a small piece of code that you can share that can reproduce this issue?

**Comment by ccordoba12** [comment]
Hey @ManlikeDBreezy, thanks for reporting. This problem is fixed in our latest version available in Anaconda (5.3.3). Please update by closing Spyder, opening the Anaconda Prompt and running there the commands mentioned in [our documentation](http://docs.spyder-ide.org/current/installation.html#updating-with-anaconda).
